kivy/uix/scatter.py

```diff
@@ -476,6 +476,8 @@ def transform_with_touch(self, touch):
             return changed
 
         angle = radians(new_line.angle(old_line)) * self.do_rotation
+        if angle:
+            changed = True
```

Comment:
I'm not very familiar, but what if the previous angle was non-zero and now it is set to zero, shouldn't we still dispatch?

Reply:
Angle is more angle diff. The transformation are cumulative. If there is no angle, you just apply identity so there is no change so changed is False. If there is a rotation, then the transformation is marked as changed.